packages/perseus/src/widgets/__tests__/definition_test.js

```diff
@@ -49,28 +49,30 @@ describe("Definition widget", () => {
         expect(container).toMatchSnapshot("first render");
     });
 
-    it("should have an open state snapshot", () => {
+    it("should have an open state snapshot", async () => {
         // Arrange
         const {container} = renderQuestion(question);
 
         // Act
         const definitionAnchor = screen.getByText("the Pequots");
-        userEvent.hover(definitionAnchor);
+        userEvent.click(definitionAnchor);
 
         // Assert
+        const tooltip = await screen.findByRole("dialog");
+        expect(tooltip).toBeVisible();
         expect(container).toMatchSnapshot("open state");
     });
 
-    it("should display the definition on hover", async () => {
```

Comment:
So do we no longer open the definition on hover? I think we have this same test on line 80.